content/docs/command-reference/import-url.md

```diff
@@ -109,8 +118,8 @@ $ dvc run -n download_data \
 ```
 
 `dvc import-url` generates an import stage
-[`.dvc` file](/doc/user-guide/dvc-files-and-directories#dvc-files) and `dvc run`
-a regular stage (in
+[`.dvc` file](/doc/user-guide/dvc-files-and-directories#dvc-files), and
+`dvc run` generates a regular stage (in
```

Comment:
Readability improvement. I think it can be changed back to original too, was just a potential enhancement.
Or we could also add 
```
..., while `dvc run` generates...
```

Reply:
Out of scope. Please roll back.

I would decide now since you already noticed this paragraph but we are going to get rid of the "import stage" term soon so this whole sentence will have to be rewritten anyway.

Reply:
BTW, appreciate the initiative! It's just that right now we want to have very small and focused PRs during this part of the process 🙂